From: Kai-Uwe Behrmann <ku.b@gmx.de>
Date: Wed, 20 Mar 2013 17:15:17 +0000 (+0100)
Subject: * [core]: fix double object release
X-Git-Url: http://www.oyranos.org/scm?p=oyranos.git;a=commitdiff_plain;h=6214cf15fafae52d67233d27462488487091d8d6

* [core]: fix double object release

... which was revealed by valgrind
---

diff --git a/src/API_generated/oyranos_devices.c b/src/API_generated/oyranos_devices.c
index 4ed5138..8cee0c8 100644
--- a/src/API_generated/oyranos_devices.c
+++ b/src/API_generated/oyranos_devices.c
@@ -1896,7 +1896,6 @@ OYAPI int  OYEXPORT
         oyConfigs_MoveIn( taxi_devices, &taxi_dev, -1 );
       }
       oyConfigs_Release( &configs_ );
-      oyOptions_Release( &options );
       oyFree_m_(ranks);
     }
 
diff --git a/src/templates/oyranos_devices.template.c b/src/templates/oyranos_devices.template.c
index 586ba36..ff555e3 100644
--- a/src/templates/oyranos_devices.template.c
+++ b/src/templates/oyranos_devices.template.c
@@ -1881,7 +1881,6 @@ OYAPI int  OYEXPORT
         oyConfigs_MoveIn( taxi_devices, &taxi_dev, -1 );
       }
       oyConfigs_Release( &configs_ );
-      oyOptions_Release( &options );
       oyFree_m_(ranks);
     }
 

